<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CazareStudenti</class>
 <widget class="QMainWindow" name="CazareStudenti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>826</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>826</width>
    <height>600</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Cazare Studenti</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="mainWindow">
   <widget class="QPushButton" name="displayTable">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>DISPLAY</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="displayWindow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>761</width>
      <height>381</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="selectTableBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="insertIntoTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>INSERT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateTable">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFromTable">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectTable_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SELECT TABLE:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearDisplay">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deselect">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>DESELECT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
